TODO

```diff
@@ -1420,6 +1420,7 @@ Features:
   - make it operate on loopback files, dissecting enough to find ESP to operate on
   - bootspec: properly support boot attempt counters when parsing entry file names
   - support --quiet in is-installed, update
+  - can we load Type#2 kernels from XBOOTLDR?
```

Comment:
You got a bit confused there. It can load type1 and 2 from both ESP and XBOOT. But it does not look at any other configuration (loader.conf, random-seed).

Reply:
Ah, fuck. Indeed. I was looking at the code and got confused. I'll readjust the text again.